src/Categories/Functor/Core.agda

```diff
@@ -28,6 +28,7 @@ record Functor (C : Category o ℓ e) (D : Category o′ ℓ′ e′) : Set (o 
   -- nice shorthands
   ₀ = F₀
   ₁ = F₁
+  resp-≈ = F-resp-≈
```

Comment:
If you like; I was just following along with what was there.

Reply:
Adding things to `.Core` should be done very rarely. There is a huge advantage to the subscripts for presentation, but not as much when moving from 8 characters to 6. Doesn't quite pull its weight.  [A meaningful but quite short name for this would be worth it, BTW. I just don't have a good candidate.]

Reply:
To me the biggest advantage as a reader was the elimination of the confusing occurrences of F when a functor (but not this one) is named F ... your mileage may vary.  Actually the canonical stdlib-style name for this should be 'cong' or '\_1-cong' in any case ...

Reply:
A long time ago, we had quite a comprehensive discussion around this sort of thing in #103 but nothing ever came of it, apparently. Maybe it's time to revisit this?

Reply:
I'd say so, but for now I took the change out of this PR.